<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>qtAppClass</class>
 <widget class="QDialog" name="qtAppClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>826</width>
    <height>479</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>qtApp</string>
  </property>
  <widget class="QLineEdit" name="eMailLineEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>20</y>
     <width>381</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QLineEdit" name="passwdLineEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>60</y>
     <width>381</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>E-Mail:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>60</y>
     <width>51</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Password:</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Your ID:</string>
   </property>
  </widget>
  <widget class="Line" name="line">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>80</y>
     <width>541</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_2">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>0</y>
     <width>541</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_3">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>10</y>
     <width>41</width>
     <height>81</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_4">
   <property name="geometry">
    <rect>
     <x>540</x>
     <y>10</y>
     <width>21</width>
     <height>81</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="genPasswdButton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>20</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="text">
    <string>Generate
Password</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="yourIdLineEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>110</y>
     <width>381</width>
     <height>20</height>
    </rect>
   </property>
   <property name="readOnly">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QLabel" name="label_4">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>140</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Recipient:</string>
   </property>
  </widget>
  <widget class="QLineEdit" name="recipientIdLineEdit">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>140</y>
     <width>381</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QPushButton" name="addButton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>140</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Add</string>
   </property>
  </widget>
  <widget class="QComboBox" name="comboBox">
   <property name="geometry">
    <rect>
     <x>80</x>
     <y>170</y>
     <width>381</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QLabel" name="label_5">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>170</y>
     <width>51</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Recipients:</string>
   </property>
  </widget>
  <widget class="QPushButton" name="startButton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>170</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Start</string>
   </property>
  </widget>
  <widget class="Line" name="line_5">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>190</y>
     <width>541</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>541</width>
     <height>20</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
  </widget>
  <widget class="Line" name="line_7">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>100</y>
     <width>41</width>
     <height>101</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="Line" name="line_8">
   <property name="geometry">
    <rect>
     <x>530</x>
     <y>100</y>
     <width>41</width>
     <height>101</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Vertical</enum>
   </property>
  </widget>
  <widget class="QPushButton" name="loginButton">
   <property name="geometry">
    <rect>
     <x>470</x>
     <y>60</y>
     <width>71</width>
     <height>23</height>
    </rect>
   </property>
   <property name="text">
    <string>Login</string>
   </property>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="qtapp.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>loginButton</sender>
   <signal>clicked()</signal>
   <receiver>qtAppClass</receiver>
   <slot>loginButton_click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>509</x>
     <y>78</y>
    </hint>
    <hint type="destinationlabel">
     <x>561</x>
     <y>73</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>genPasswdButton</sender>
   <signal>clicked()</signal>
   <receiver>qtAppClass</receiver>
   <slot>genPasswdButton_click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>527</x>
     <y>35</y>
    </hint>
    <hint type="destinationlabel">
     <x>590</x>
     <y>38</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>addButton</sender>
   <signal>clicked()</signal>
   <receiver>qtAppClass</receiver>
   <slot>addButton_click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>496</x>
     <y>150</y>
    </hint>
    <hint type="destinationlabel">
     <x>592</x>
     <y>150</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>startButton</sender>
   <signal>clicked()</signal>
   <receiver>qtAppClass</receiver>
   <slot>startButton_click()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>509</x>
     <y>182</y>
    </hint>
    <hint type="destinationlabel">
     <x>632</x>
     <y>183</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>loginButton_click()</slot>
  <slot>genPasswdButton_click()</slot>
  <slot>startButton_click()</slot>
  <slot>addButton_click()</slot>
 </slots>
</ui>
